app/serializers/atmosphere/virtual_machine_flavor_serializer.rb

```diff
@@ -5,7 +5,19 @@ class VirtualMachineFlavorSerializer < ActiveModel::Serializer
     attributes :id, :flavor_name,
       :cpu, :memory, :hdd, :hourly_cost,
       :compute_site_id, :id_at_site,
-      :supported_architectures, :active
+      :supported_architectures, :active, :cost_map
+
+    private
+
+    # Deprecated: returns max hourly cost
+    def hourly_cost
+      object.virtual_machine_flavor_os_families.max_by(&:hourly_cost).hourly_cost
+    end
+
+    # Returns a full cost map for this flavor (depending on os_family)
+    def cost_map
+      object.cost_map
+    end
```

Comment:
Similar situation as above. You don't need to define this method here. If it exists in serialized object it will be used without a problem.